phonebuddies/PhoneNumberParser.py

```diff
@@ -0,0 +1,52 @@
+import re
+import phonenumbers
+class PhoneNumberParser:
+    @staticmethod
+    def parse_to_valid_whatsapp(phone_number):
+        print (phone_number)
+        # Remove any non-digit characters from the phone number
+        no_whitespace = phone_number.replace(' ','').replace('+','')
```

Comment:
Remove whitespace with a regex to get all characters, not just those in the tests (including invisible chars)

Reply:
Actually I think you can just remove this given line 9 below